spec/paperclip/plural_cache_spec.rb

```diff
@@ -3,34 +3,35 @@
 describe 'Plural cache' do
   it 'caches pluralizations' do
     cache = Paperclip::Interpolations::PluralCache.new
-    word = "box"
+    symbol = :box
 
-    word.expects(:pluralize).returns("boxes").once
-
-    cache.pluralize(word)
-    cache.pluralize(word)
+    first = cache.pluralize_symbol(symbol)
+    second = cache.pluralize_symbol(symbol)
+    expect(first).to equal(second)
   end
 
   it 'caches pluralizations and underscores' do
+    class BigBox ; end
     cache = Paperclip::Interpolations::PluralCache.new
-    word = "BigBox"
-
-    word.expects(:pluralize).returns(word).once
-    word.expects(:underscore).returns(word).once
+    klass = BigBox
 
-    cache.underscore_and_pluralize(word)
-    cache.underscore_and_pluralize(word)
+    first = cache.underscore_and_pluralize_class(klass)
+    second = cache.underscore_and_pluralize_class(klass)
+    expect(first).to equal(second)
   end
 
   it 'pluralizes words' do
     cache = Paperclip::Interpolations::PluralCache.new
-    word = "box"
-    assert_equal "boxes", cache.pluralize(word)
+    symbol = :box
+
+    expect(cache.pluralize_symbol(symbol)).to eq("boxes")
   end
 
-  it 'pluralizes and underscore words' do
+  it 'pluralizes and underscore class names' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.